File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int n1= s.length();
        int n2 = t.length();

        if(n1 != n2) return false;

        int count[26] = {0};

        for(int i=0; i<n1 ;i++) {
            count[s[i] - 'a']++ ;
            count[t[i] - 'a']-- ;
        }    
        for(int value : count) {
            if(value != 0) {
                return false;
            }
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        n1 = len(s)
        n2 = len(t)

        if(n1 != n2): return False

        count = [0] * 26

        for i in range(n1) :
            count[ord(s[i]) - ord('a')] += 1
            count[ord(t[i]) - ord('a')] -= 1
        
        for value in count:
            if value != 0:
                return False
        return True